firestore/src/common/firestore.cc

```diff
@@ -226,6 +241,7 @@ void Firestore::DeleteInternal() {
   if (!internal_) return;
 
   App* my_app = app();
+  const std::string& database_id = internal_->database_id().database_id();
```

Comment:
TODO: Check why is this returning emoty string instead of "(default)".

Reply:
Terminated the firestore first, which wipes out the database ID, and then referering to database_id will return empty string, as it is already deleted.